app/istqb-ct-genai-complete-guide/istqb-ct-genai-complete-guide.css

```
@@ -0,0 +1,1251 @@
+                font-family: var(--font-mono, 'Space Mono', monospace);
+                font-size: 1rem;
+                color: var(--color-text-muted);
+                text-transform: uppercase;
+                letter-spacing: 0.1em;
+            }
+            .code-block pre {
+                font-family: var(--font-mono, 'Space Mono', monospace);
+                font-size: 1rem;
+                line-height: 1.7;
+                white-space: pre-wrap;
+                word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)